src/clients/BundleDataClient/utils/SuperstructUtils.ts

```diff
@@ -10,14 +10,15 @@ import {
   pattern,
   boolean,
   defaulted,
+  union,
   type,
 } from "superstruct";
 import { BigNumber } from "../../../utils";
 
 const PositiveIntegerStringSS = pattern(string(), /\d+/);
 const Web3AddressSS = pattern(string(), /^0x[a-fA-F0-9]{40}$/);
 
-const BigNumberType = coerce(instance(BigNumber), string(), (value) => {
+const BigNumberType = coerce(instance(BigNumber), union([string(), number()]), (value) => {
```

Comment:
I think we have a BigNumber SS type elsewhere in the org. Either in the relayer or the SDK.